src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSProject_IWorkspaceProjectContext.cs

```diff
@@ -120,12 +121,16 @@ public void SetOptions(string commandLineForOptions)
             ExecuteForegroundAction(() =>
             {
                 var commandLineArguments = SetArgumentsAndUpdateOptions(commandLineForOptions);
-                if (commandLineArguments != null)
-                {
-                    // some languages (e.g., F#) don't expose a command line parser and this might be `null`
-                    SetRuleSetFile(commandLineArguments.RuleSetPath);
```

Comment:
This call would cause us to immediately throw away and reparse our options.  I'm trying to avoid that.